docs/serving/services/pvc.md

```diff
@@ -0,0 +1,46 @@
+# Permanent data storage
+
+You can provide permanent data storage for Knative Services by configuring `PersistentVolumeClaims` (PVCs).
```

Comment:
Nit: I'd describe this as "shared filesystem storage".  Given the description below, you'd probably also want to highlight that your volume claim will need an `accessMode` of `ReadWriteMany` if you want read-write access.  If you want read-only shared data, you can set the `readOnly` property of the `volume` entry to `true` and use `ReadOnlyMany`.

`ReadWriteOnce` and `ReadWriteOncePod` will not work with Knative autoscaling.

Reply:
Great feedback, thanks!

Reply:
I also provided some naming alternatives above. Besides the access mode, depending on the volume type users may have to adjust ksvc security context wrt to file related properties eg. FSGroup.